lib/SIL/Utils/MemAccessUtils.cpp

```diff
@@ -92,7 +92,6 @@ AccessedStorage::AccessedStorage(SILValue base, Kind kind) {
     value = base;
     break;
   case Stack:
-    assert(isa<AllocStackInst>(base));
```

Comment:
I'd like to keep the assertion here but, it would require duplicating the code in `visitClassAccess` and I don't think it's worth it just for the assertion. WDYT?